BHoMUpgrader42/Converter.cs

```diff
@@ -47,7 +48,57 @@ public Converter() : base()
         /***************************************************/
 
 
+        public static Dictionary<string, object> UpgradeUniformLoadSet(Dictionary<string, object> oldVersion)
+        {
+            if (oldVersion == null)
+                return null;
+
+            Dictionary<string, object> newVersion = new Dictionary<string, object>();
+
+            newVersion["_t"] = "BH.oM.Structure.Loads.UniformLoadSet";
+            newVersion["Name"] = oldVersion["Name"];
+            newVersion["CustomData"] = oldVersion["CustomData"];
+            newVersion["BHoM_Guid"] = oldVersion["BHoM_Guid"];
+            newVersion["Tags"] = oldVersion["Tags"];
+            newVersion["Fragments"] = oldVersion["Fragments"];
+
+            object outObject;
+            
+            if ( oldVersion.TryGetValue("Loads", out outObject) )
+            {
+                Dictionary<string, object> oldLoadDict = outObject as Dictionary<string, object>;
+
+                oldVersion.TryGetValue("Loadcases", out outObject);
 
+                if (outObject != null)
+                {
+                    Dictionary<string, object> oldLoadcaseDict = outObject as Dictionary<string, object>;
+                    List<string> names = oldLoadDict.Keys.ToList();
+
+                    newVersion["Loads"] = names.Select(x => new Dictionary<string, object>()
+                    {
+                        { "_t", "BH.oM.Structure.Loads.UniformLoadSetRecord" },                   
+                        { "Name", x },
+                        { "Loadcase", oldLoadcaseDict[x] },
+                        { "Load", oldLoadDict[x] },
+                }).ToList();
```

Comment:
This line needs indenting one extra indent to line up correctly with the bits above it.